cdap-app-fabric-tests/src/test/java/io/cdap/cdap/internal/app/services/http/handlers/WorkflowStatsSLAHttpHandlerTest.java

```diff
@@ -204,23 +206,25 @@ public void testStatistics() throws Exception {
                             System.currentTimeMillis(),
                             "99");
     response = doGet(request);
-    Assert.assertEquals(HttpResponseStatus.OK.code(), response.getResponseCode());
-    Assert.assertTrue(
-      response.getResponseBodyAsString().startsWith("There are no statistics associated with this workflow : "));
+    // no apps found
+    Assert.assertEquals(HttpResponseStatus.NOT_FOUND.code(), response.getResponseCode());
```

Comment:
Why are expectations changed here?

Reply:
@jingtaowu3 can you pls take this one

Reply:
reverted changes in the test case here. Did a small fix to keep the behavior unchanged. Now it will not throw a NotFoundException if the workflow does not exist.